guides/P2PTunnels.md

```diff
@@ -0,0 +1,48 @@
+# How to Use P2P Tunnels in Applied Energistics 2
+AE2 (short for Applied Energistics 2) is a great mod for storing your items, autocrafting and much more. However once you'll get a big ME system going, you'll start running into some issues with **channels**.
+
+## Channels 
+Most of the things you can connect to your network (interfaces, ME Drives, ME chests, terminals, buses, crafting multiblocks etc.) use up a channel each, and channels are provided from your ME controller multiblock. These blocks also do transmit channels themselves. You can carry 8 channels with most cables, but dense cables allow you to carry 32.
+
+![2022-06-18_10 54 16](https://user-images.githubusercontent.com/100072467/174430656-b3ce42a5-1a8f-4c33-8601-edf96c5d9e14.png)
```

Comment:
could you name the file descriptively, and link to it via relative links rather than uploaded content.

Reply:
yeah sorry, gonna fix it